src/main/java/com/google/devtools/build/lib/remote/blobstore/http/AbstractHttpHandler.java

```diff
@@ -34,6 +35,7 @@
 abstract class AbstractHttpHandler<T extends HttpObject> extends SimpleChannelInboundHandler<T>
     implements ChannelOutboundHandler {
 
+  private static final String USER_AGENT_VALUE = "bazel " + BlazeVersionInfo.instance().getVersion();
```

Comment:
```suggestion
  private static final String USER_AGENT_VALUE = "Bazel/" + BlazeVersionInfo.instance().getVersion();
```


https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent

Reply:
Thanks! I'd prefer to stay with 'bazel' rather than 'Bazel' as the first one is used for gRPC cache already. Updated accordingly.